modules/dnn/test/test_layers_1d.cpp

```diff
@@ -245,4 +245,48 @@ INSTANTIATE_TEST_CASE_P(/*nothing*/, Layer_Elemwise_1d_Test, Combine(
 /*operation*/           Values("div", "prod", "max", "min", "sum")
 ));
 
+// WORKING FOR 1D
+typedef testing::TestWithParam<tuple<int>> Layer_Permute_Test;
+TEST_P(Layer_Permute_Test, Accuracy)
```

Comment:
Comment doesn't correlate with the test name.

In test reports we don't not see comments of the source code.

Reply:
Removed

Reply:
"Accuracy" is too wide and should be narrowed down to "Accuracy_0d_1d" for example.